src/components/map/hooks/use-trigger-camera-move-effect.tsx

```diff
@@ -40,7 +41,13 @@ export const useTriggerCameraMoveEffect = (
 ) => {
   const {height: bottomSheetHeight} = useBottomSheet();
   const {minHeight: tabBarMinHeight} = useBottomNavigationStyles();
-  const paddingBottom = bottomSheetHeight - tabBarMinHeight;
+  var paddingBottom = bottomSheetHeight - tabBarMinHeight;
+
+  paddingBottom =
+    // Workaround for Android when font and screen size changes!
+    Platform.OS == 'android' && PixelRatio.get() > 3.0
+      ? paddingBottom / PixelRatio.get()
+      : paddingBottom;
```

Comment:
This looks kind of hacky 🤔 Is there something special about pixel ration being > 3, or could we solve it in a more generic way? Would it work to always divide by pixel ratio, regardless of amount?

But if this is the best solution, this might be a more readable way to write this 😊

```suggestion
  let paddingBottom = bottomSheetHeight - tabBarMinHeight;

  // Workaround for Android when font and screen size changes
  if (Platform.OS == 'android' && PixelRatio.get() > 3.0) {
    paddingBottom = paddingBottom / PixelRatio.get();
  }
```

Reply:
Yes, it looks hacky! Indeed! but after many tests, I realize that Android breaks when PixelRatio is bigger than 3.0, I could show you, and maybe together we could find something better :)

Reply:
We looked at it today and got it working without the "magic number".